src/components/policy/src/cache_manager.cc

```diff
@@ -1187,6 +1187,10 @@ bool CacheManager::IsPredataPolicy(const std::string& app_id) {
       pt_->policy_table.app_policies_section.apps[app_id];
 
   policy_table::Strings res;
+
+  sort(pre_data_app.groups.begin(), pre_data_app.groups.end());
+  sort(specific_app.groups.begin(), specific_app.groups.end());
+
   std::set_intersection(pre_data_app.groups.begin(),
```

Comment:
@anosach-luxoft in windows platform there is an assert in the implementation of the standard library and is required to be sorted. By the way it is done in policy_helper.cc now I applied it here too.